ArithmeticToolsTests/IsPowerOfTwoTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  IsPowerOfTwoTests.swift
+//  ArithmeticTools
+//
+//  Created by Brian Heim on 12/11/16.
+//  Copyright © 2016 James Bean. All rights reserved.
+//
+
+import XCTest
+@testable import ArithmeticTools
+
+class IsPowerOfTwoTests: XCTestCase {
+    
+    func testIntIsPowerOfTwoTrue() {
+        XCTAssertTrue(1.isPowerOfTwo)
+        XCTAssertTrue(2.isPowerOfTwo)
+        XCTAssertTrue(4.isPowerOfTwo)
+        XCTAssertTrue(1024.isPowerOfTwo)
+        // XCTAssertTrue((-1).isPowerOfTwo)
```

Comment:
XCode was giving me parse errors without them..
And oh duh, i forgot that 2^-1 isn't -2.